hbshelpers/sdkAssetUrl.js

```diff
@@ -24,17 +30,21 @@ module.exports = function sdkAssetUrl(branch, locale, assetName) {
     parsedBranch = `dev/${branch.replace(/\//g, '-')}`;
   }
 
-  const isPreReleaseBranch = 
+  const isPreReleaseBranch =
     RELEASE_BRANCH_REGEX.test(branch) || HOTFIX_BRANCH_REGEX.test(branch);
   const isI18nFeatureBranch = I18N_FEATURE_BRANCH_REGEX.test(branch);
   const isDevelopBranch = branch === 'develop';
-  const isLocalizationSupported = 
-    (isReleasedBranch || isPreReleaseBranch || isI18nFeatureBranch || isDevelopBranch) && 
+  const isLocalizationSupported =
+    (isReleasedBranch || isPreReleaseBranch || isI18nFeatureBranch || isDevelopBranch) &&
     !(locale.startsWith('en') || assetName === 'answers.css') ;
-  
+
   const parsedAssetName = isLocalizationSupported ?
     `${locale}-${assetName}` :
     assetName;
 
-  return `https://assets.sitescdn.net/answers/${parsedBranch}/${parsedAssetName}`;
+  const domain = cloudRegion === 'eu'
```

Comment:
can you create a const for 'eu' to improve readability? Maybe one for 'us' as well.

Reply:
sure